### PR TITLE
[WFCORE-5292] Remove Java EE References from the i18n strings

### PR DESCRIPTION
**Pull request by yersan**
Jira issue: https://issues.redhat.com/browse/WFCORE-5292

@RanabirChakraborty review if you have a chance, thanks!

### CODE REVIEW

File: server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties [hunk 1]
```diff
@@ -257,7 +257,7 @@ server-set-restart-required=Puts the server into a restart-required mode.
 write-config=An operation to force the server to write its config file, without making any actual config change.
 
 # Deployments
-deployment=A deployment represents anything that can be deployed (e.g. an application such as EJB-JAR, WAR, EAR, any kind of standard archive such as RAR or JBoss-specific deployment) into a server.
+deployment=A deployment represents anything that can be deployed (e.g. an application such as Jakarta EJB-JAR, WAR, EAR, any kind of standard archive such as RAR or JBoss-specific deployment) into a server.
```
Comment (yersan) [inline server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties:260]: I'm not 100% sure if that is the correct one "EJB-JAR" vs "Jakarta EJB-JAR"

Maybe "Jakarta Enterprise Beans-JAR" instead?

**Review by RanabirChakraborty — APPROVED** [review]
Checked and it looks good to me.

File: server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties (hunk 1 above)
Comment (RanabirChakraborty) [reply server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties:260]: `Jakarta Enterprise Beans-JAR` will be a better choice I think.

Comment (tadamski) [reply server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties:260]: I agree, "Jakarta Enterprise Beans-JAR" seems good.

**Review by jamezp — APPROVED** [review]
This looks good to me.

**Pull request merged**